test/test_bin/test_check_mta_sts.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import boto3
+from botocore.exceptions import 
```

Comment:
Looks like the interpreter is complaining about this line since it expects what you want to import after the import keyword 👀 

Maybe you meant?

```suggestion
import botocore.exceptions
```